ios-release-build.md

```diff
@@ -0,0 +1,198 @@
+
+1. Click '+' to create a new Bundle ID.
+
+1. Enter an app name and select `Explicit App ID` and enter an ID.
+
+1. Select any services, then click `Continue`.
+
+1. On the next page, confirm the details and click `Register` to register your
+   Bundle ID.
+
+### Create an iTunes Connect application record
+
+The next step is to register your app on iTunes Connect. Follow these steps:
+
+1. Open [iTunes Connect][itunesconnect] in your browser.
+
+1. On the iTunes Connect landing page, click `My Apps`.
+
+1. Click the '+' sign in the top-left corner of the `My Apps` page, then select
+   `New App`.
+
+1. Fill in your app details on the form that appears. In the `Platforms`
+   section, ensure that iOS is checked. Leave tvOS unchecked. Click `Create`.
+
+1. Navigate to the application details for your app and select `App
+   Information` from the sidebar.
+
+1. In the `General Information` section, select the Bundle ID you registered
+   above.
+
+For a detailed overview, see [Creating an iTunes Connect Record for an
+App][itunesconnect_guide_register] in the iTunes Connect Developer Guide.
+
+## Reviewing the Xcode project properties
+
+When you create a new app, a default Xcode workspace is created at `<app
+dir>/ios/Runner.workspace`. Before releasing, you should review your project
+settings in Xcode and verify the values are correct.
+
+To view the settings, select the `Runner` project in the Xcode project
+navigator. Then, in the main view sidebar, select the `Runner` target and
+select the `General` tab and verify the following settings:
+
+In the `Identity` section:
+
+  * `Display Name`: the name of the app to be displayed on the home screen and
+    elsewhere.
+  * `Bundle Idenfitifer`: the App ID you registered above on iTunes Connect.
+
+In the `Deployment Info` section:
+
+  * `Deployment Target`: Set to `8.0`. If your app includes Objective-C or
+    Swift code that makes use of APIs that were unavailable in iOS 8, update
+    this value appropriately.
+
+## Adding an App Icon
+
+When a new Flutter app is created, it has a default App Icon. To customize this
+icon:
+
+1. Review the [iOS App Icon][appicon] guidelines for icon design.
+
+1. In the Xcode project navigator, select `Assets.xcassets` in the `Runner`
+   folder. Update the icons with your app icons.
+
+1. To verify the icon has been replaced, run your app using `flutter run`
+and inspect the icon.
+
+## Signing the app
+
+To configure signing, select the `Runner` project in the Xcode project
+navigator. Then, in the main view sidebar, select the `Runner` target and open
+the `General` tab.
+
+In the `Signing` section, check the `Automatically manage signing` checkbox,
+then select your registered development team from the `Team` pop-up menu.
+
+For a detailed overview of app signing, see [Maintaining Your Signing
+Identities and Certificates][appsigning] in Apple's App Distribution Guide.
+
+## Creating a build archive
```

Comment:
Isn't this a substep of the previous `## Register your app on iTunes Connect` section?